src/foundation.cs

```diff
@@ -1309,6 +1313,26 @@ interface NSCoder {
 		[iOS (9,0)][Mac (10,11)]
 		[NullAllowed, Export ("error", ArgumentSemantic.Copy)]
 		NSError Error { get; }
+
+		[Watch (7,0), TV (14,0), Mac (10,16), iOS (14,0)]
+		[Export ("decodeArrayOfObjectsOfClass:forKey:")]
+		[return: NullAllowed]
+		NSObject[] DecodeArrayOfObjects (Class cls, string key);
```

Comment:
I believe the guidelines say that parameters shouldn't be abbreviated:

```suggestion
		NSObject[] DecodeArrayOfObjects (Class @class, string key);
```

Reply:
I'm always ambivalent between _better_ or original parameter names. Been thinking of an xtro rules to check them - but it would be too noisy (and in general less good).